src/Core/System/SystemConfig/Event/BeforeSystemConfigChangedEvent.php

```diff
@@ -13,8 +13,7 @@ class BeforeSystemConfigChangedEvent extends Event
      */
     public function __construct(
         private readonly string $key,
-        /** @deprecated tag:v6.7.0 - Will be natively typed */
-        private $value,
+        private mixed $value,
```

Comment:
I did the same change in my PR. I would not use "mixed", as we only allow scalar values and arrays as values for the system config, don't we? 
so:
```suggestion
        private array|bool|float|int|string|null $value,
```

Reply:
makes sense